recipe/0001-removing-gcc-pragma-windows.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/mkl_random/src/mkl_distributions.cpp b/mkl_random/src/mkl_distributions.cpp
```

Comment:
Can you please add a comment to this patch with some background? (E.g., who provided the information that `__pragma(ivdep)` is the right thing to do.)

Reply:
I added a few notes to the patch itself. I am not sure that it fully satisfies your question. Let me know if there is more I should add.